### ISSUE TITLE
Version 1.0

### ISSUE THREAD

**Issue by Razican** [achanda/ipnetwork#164]
As far as I can tell, the public API of this crate has stayed backwards compatible for quite a few versions already. This means that releasing a 1.0 version in the current form would allow to add new features (in 1.x) versions without sacrificing backwards compatibility.

This, in turn, would make it easier for crates such as `diesel` and `sqlx` to keep an up to date `ipnetwork` dependency without having to release a new version themselves whenever a new `ipnetwork` version is released.

Is there something preventing the release of a 1.0 version?

**Comment by Alextopher** [comment]
I agree, looking into this project from the outside I see a few potential blockers:

- [ ] Decide how to handle Ipv{4,6}Network::size() on maximum networks #130 #178 
- [ ] Related : decide whether or not to keep `NetworkSize`
- [ ] Whether or not to include subtraction / discontinuous network segments #126 
- [ ] Clearly define behavior through the use of doc tests
